### DISCUSSION TITLE
演示网站打不开

### DISCUSSION THREAD

**Discussion by elarbor** [armomu/vue-material-admin#39] [General]
能不能部署GitHub pages 啊。你这个打不开
![image](https://github.com/user-attachments/assets/75eaaeb1-5bb5-45a6-b3a0-78bff81d1efa)

**Comment by armomu** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    要梯子！不行你就下载到本地跑
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                894620576
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
 
    

  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2024年10月25日 09:47
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              [armomu/vue-material-admin] 演示网站打不开 (Discussion #39)
        
      
    
  
  
能不能部署GitHub pages 啊。你这个打不开
image.png (view on web)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Reply by elarbor** [reply]
这是怎么回事啊
![image](https://github.com/user-attachments/assets/bf792205-9d19-40b7-843a-cb086d96a6ef)


是需要启动配套的后端？

**Reply by armomu** [reply]
打开文件 .env.development
改成下面这样就行了，已经配套了后端apifox已经模拟了全部接口了

# VITE_API = http://localhost:8085
VITE_API = https://apifoxmock.com/m1/5061937-4723200-default

**Reply by elarbor** [reply]
好了
![image](https://github.com/user-attachments/assets/38d28069-124c-425f-8471-a20070e851d9)




真优秀的项目啊


建议你在https://github.com/armomu/vue-material-admin?tab=readme-ov-file#%E5%89%8D%E7%AB%AF 注明
